assets/scss/components/citadel/forms/_forms.scss

```diff
@@ -57,7 +57,14 @@
             position: right $form-success-icon-right center;
             repeat: no-repeat;
         }
-        padding-right: $form-success-input-paddingRight;
+        padding-right: $form-success-input-paddingRight;  
+    }
+    .form-input[data-label="Credit Card Number"] {
```

Comment:
Do we need these changes only for two inputs?

Reply:
I checked where it would be possible to use these changes, but I did not find suitable inputs that require these changes.

Reply:
will this label be different on different languages?

Reply:
If you mean the name label "Credit Card Number" and whether the name will be translated. No, this label won’t  be different.

Reply:
@yurytut1993 
You're right label will different, so I replaced the data-label on name.